### PR TITLE
Seperate plugins from vib codebase

### PR DESCRIPTION
**Pull request by axtloss**
This separates our builtin plugins (except for `shell`) from the vib codebase, making vib more versatile and lighter as it does not force the plugins we use in vanilla os.
As part of the separation it is possible to install system wide vib plugins into `/usr/share/vib/plugins` which can be used in any project but will **not** be preferred over local plugins.

This is part of an ongoing effort to rework the vib api to work with more programming languages.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,8 +19,12 @@ jobs:
...
         go get ./...
-        go build -v -trimpath ./...
-        
\ No newline at end of file
+        make build
+	
+    - name: Build plugins
+      run: |
+      	go get ./...
+	make build-plugins
```
Comment (matbme) [inline .github/workflows/build.yml:30]: I think this spacing can cause problems when running the workflow

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -20,10 +20,18 @@ jobs:
     - name: Build
       run: |
         go get ./...
-        go build -o vib -trimpath
+        make build
+
+    - name: Build plugins
+      run: |
+        go get ./...
+	make build-plugins
```
Comment (matbme) [inline .github/workflows/release.yml:28]: Same here

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -20,10 +20,18 @@ jobs:
...
+	tar cvf plugins.tar.xz build/plugins
 
     - name: Upload a Release Asset
       if: github.repository == 'Vanilla-OS/Vib'
       uses: softprops/action-gh-release@v2
       with:
-        files: vib
+        files: |
+	  build/vib
+	  plugins.tar.xz
```
Comment (matbme) [inline .github/workflows/release.yml:37]: And here

**Review by matbme — APPROVED** [review]

**Review by mirkobrombin — APPROVED** [review]
Looks good to me

**Pull request merged**